dev/moclojer/build.clj

```diff
@@ -2,26 +2,31 @@
   (:require [clojure.data.json :as json]
             [clojure.java.io :as io]
             [clojure.string :as string]
-            [clojure.tools.build.api :as b]
-            [moclojer.helper :as helper]))
+            [clojure.tools.build.api :as b]))
 
 (def lib 'moclojer/moclojer)
 (def class-dir "target/classes")
 (def uber-file "target/moclojer.jar")
+(def moclojer-version
```

Comment:
this way when running (server up) the software it will always look for the `VERSION_PREFIX` file, i.e., when running the binary version (generated by `graalvm/native-image`) it will not find the file

here is a test saying:
https://github.com/avelino/moclojer/runs/5537930745?check_suite_focus=true#step:7:60

I put the version declaration in a package [helper](https://github.com/avelino/moclojer/blob/main/src/moclojer/helper.clj#L3)

Reply:
On this branch, everything is running ok
https://github.com/avelino/moclojer/runs/5547898585?check_suite_focus=true#step:5:29


This `moclojer-version` is at `moclojer.build`, not in `moclojer.core` as in your link

- `moclojer.build` will not be included/executed in the final jar/binary
- `moclojer.build` will only be called at devtime, so it is OK to expect that it depends on repository files

Reply:
used here: https://github.com/avelino/moclojer/blob/main/src/moclojer/core.clj#L43

Reply:
done